policyengine_us/parameters/gov/states/ny/tax/income/credits/cdcc/max.yaml

```diff
@@ -26,7 +26,9 @@ metadata:
   threshold_unit: dependent
   amount_unit: currency-USD
   amount_period: year
-  label: NY CDCC max amount
+  label: New York CDCC max amount
   reference:
+    - title: New York Instructions for Form IT-201 Line 64
```

Comment:
Wasn't able to find any CDCC specific tax form references to parameter values

Reply:
I have filed an issue for consolidation with the legal code to prioritize value updates in this PR